src/lib/factories/listing.ts

```diff
@@ -71,6 +71,12 @@ const defaults: ListingType = {
   range: 153,
   rangeExtraUrban: 100,
   rangeUrban: 53,
+  chargePort: ["type-1"],
```

Comment:
are those values predefined and fetched from the backend? shouldn't this be added to reference data too?
The same goes for the fast charge port

Reply:
I may have made a mistake here, the line you reviewed should just be a simple string I think `"type-1"`, as when we're sending the form it sends a string. When prefetching it gets an array of charge ports: `["type-1", "type-2"]`
Correct me if I'm wrong